splunk_add_on_ucc_framework/schema/schema.json

```diff
@@ -14,7 +14,7 @@
         },
         "type": {
           "type": "string",
-          "enum": ["singleSelectSplunkSearch"]
+          "enum": ["text", "checkbox", "singleSelect", "radio", "singleSelectSplunkSearch"]
```

Comment:
How did it work before? If only those 5 elements are allowed - then we should make sure only those 5 are allowed.

Reply:
I'll need to research this more on my own. It's fine for this PR.